### ISSUE TITLE
Nameserver issue

### ISSUE THREAD

**Issue by xiaoyong-wang** [litespeedtech/cyberpanel#232]
When i point a domain to the nameservers of my cyberpanel server, it does not work.
For example:
https://cyrex.services points to my cyberpanel servers nameservers, and the page wont load

**Comment by usmannasir** [comment]
Did you configure default nameservers?

**Comment by usmannasir** [comment]
Fixed some bugs here https://github.com/usmannasir/cyberpanel/commit/a9b6f298d7ece0539dc22a92c0631c61e4dc648e

You can get this fix from v1.9.4 branch using 

```
sh <(curl https://raw.githubusercontent.com/usmannasir/cyberpanel/v1.9.4/preUpgrade.sh || wget -O - https://raw.githubusercontent.com/usmannasir/cyberpanel/v1.9.4/preUpgrade.sh)
```

If you have already configured default nameservers do
```
rm /home/cyberpanel/defaultNameservers
```

Then configure default nameservers again. Also make sure that NS domains have NS configured properly.

I've checked, everything else seems to be working.
